README.md

```diff
@@ -24,7 +24,7 @@ MIGRATION: Make sure that your webpack 4 build does not print deprecation warnin
 
 Here are a few things that were removed but did not have deprecation warnings in v4:
 
-- IgnorePlugin and BannerPlugin must now be passed an options object.
+- IgnorePlugin and BannerPlugin must now be passed only one argument.
```

Comment:
```suggestion
- IgnorePlugin and BannerPlugin must now be passed only one argument that can be object or string or function
```

I think this might be accurate cause we need to mention the type of data. I know it might be present in the docs but keeping here might be usefull.